Add SSL_DOMAIN environment variable for custom Common Name

michaelservidio opened the pull request:
Create environment variable `SSL_DOMAIN` (optionally set and pulled from value in `.env`) which will be used in `server.pem` creation. 

Allows self-signed certificates to have a custom Common Name (instead of localhost, which would still be the default).

Use case:
This allowed me to view my local application at https://mydomain.test by setting `SSL_DOMAIN=*.mydomain.test` in `.env`

ryoluo commented on stubs/nginx.stub:
```diff
@@ -7,11 +7,12 @@
         networks:
-            - sail
\ No newline at end of file
+            - sail
```
Do not insert final new line.

michaelservidio replied:
Changed .editorconfig: `insert_final_newline = false`

ryoluo reviewed (changes requested):
Thank you for PR!
Could you update README to explain new env var?

michaelservidio commented:
For Chrome to accept the certificate as valid, it required a subjectAlternativeName to be set. Added another variable, `SSL_ALT_NAME`, to specify (e.g. `SSL_ALT_NAME=DNS:localhost,DNS:mydomain.test`).

Updated `README.md` to explain both variables.
